package.json

```diff
@@ -22,7 +22,8 @@
     "build": "node build.js",
     "build:dev": "RUNTIME_API_HOST='https://zerobase-api-stage.herokuapp.com' RUNTIME_ENV=dev node build.js",
     "start:dev": "RUNTIME_ENV=dev node devServer.js",
-    "serve": "http-server ./public"
+    "serve": "http-server ./public",
+    "start": "node devServer.js"
```

Comment:
No need to add this. In controller.ts uncomment out lines 1 -20. Then in app.json add RUNTIME_API_HOST to env